packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -496,7 +490,7 @@ describe('TokensController', () => {
         address: '0x02',
         decimals: 2,
         image:
-          'https://static.metafi.codefi.network/api/v1/tokenIcons/11155111/0x02.png',
+          'https://static.cx.metamask.io/api/v1/tokenIcons/11155111/0x02.png',
```

Comment:
I'm unable to access these static files, but I'm also unable to access them through old links. Is this expected? will these links be activated later?